### PR TITLE
average_win_percentage

### PR DESCRIPTION
**Pull request by gabichuelas**
@rrabinovitch @evilaspaas1 @sciencefixion 
Another team stats method! Did some editing of existing helper methods for this one :)

### CODE REVIEW

File: test/stat_tracker_test.rb [hunk 1]
```diff
@@ -269,16 +269,17 @@ def test_worst_season_by_team_id
 
   def test_worst_season_by_team_id_expanded
     skip
+    # THIS TEST IS BROKEN SOMEHOW
```
Comment (gabichuelas) [inline test/stat_tracker_test.rb:272]: Disregard this; it's unrelated to the method this PR is for, and I will address this `worst_season` method in a later PR.

File: test/stat_tracker_test.rb [hunk 2]
```diff
@@ -269,16 +269,17 @@ def test_worst_season_by_team_id
...
     # this is the only test that uses full csv, and
-    # it's noticeably slower.
+    # it's a little noticeably slower.
```
Comment (rrabinovitch) [inline test/stat_tracker_test.rb:274]: oh interesting, does using an expanded fixture not work with this one? also curious about how you were able to identify what the expected value here is if using the full csv's

**Review by rrabinovitch — APPROVED** [review]
nice breakdown of methods into reusable helper methods

**Review by araspaas — APPROVED** [review]
looks good to me.

File: test/stat_tracker_test.rb (hunk 2 above)
Comment (gabichuelas) [reply test/stat_tracker_test.rb:274]: @rrabinovitch I actually changed this comment in a later version I will push up later, but I realized that this is not the reason the test was slower. The reason I know this is because when I run the spec harness, that's using the real csvs so if i t was slower that's not why. Must be because there's a method somewhere that is taking particularly longer. Probably whichever method hasn't been refactored in to helpers. 

TLDR; not a big deal at all. 

As for identifying expected values, I read the errors on the spec harness, and in some cases you can see what their querying values were, and what they expected to get as an answer, assuming you're using the original csv's. This test was written because I was playing around with that concept. Need to figure out what's wrong with it still.

**Review by sciencefixion — APPROVED** [review]
